tests/nested/manual/muinstaller-core/task.yaml

```diff
@@ -17,13 +17,34 @@ environment:
   NESTED_ENABLE_TPM/plain: false
   NESTED_ENABLE_SECURE_BOOT/plain: false
 
+  # default (unencrypted) case
+  NESTED_ENABLE_TPM: false
+  NESTED_ENABLE_SECURE_BOOT: false
+
```

Comment:
You could remove the "plain" ones I think.

Reply:
I think it probably would be better to run the various test case with encryption on, right? And then leave this `plain` case to cover the unencrypted case.

Reply:
Agreed, then these two should be set to true and  maybe remove the "encrypted" case as it will be covered by the "install*" cases.

Reply:
Done in a881fd21b90f155d171b88d80c05e2004f975f36.